### ISSUE TITLE
About the running results 

### ISSUE THREAD

**Issue by XXRPKU** [LWL-cpu/SCPRG-master#2]
Hello, thanks for you interesting work! However, I cannot reproduce the results of your paper using the random seeds and hyperparameters in your bash file run_rams_base.sh. And it seems I can't get the same result with the same seed 1111 for different runs on different GPUs.  
One of my results are as follows:

for span F1
dev
Precision: 44.0585 Recall: 49.0086 F1: 46.4019
test
Precision: 46.5645 Recall: 51.8437 F1: 49.0625

for head F1
dev
Precision: 50.4113 Recall: 56.0752 F1: 53.0927
test
Precision: 52.7929 Recall: 58.7782 F1: 55.6250

Is it possible to see the same results every run? Is there a method to stably train and reduce the performance variance?

**Comment by LWL-cpu** [comment]
Thank you for your attention! We report our best result in our paper, and part of our model checkpoint are provided in https://drive.google.com/drive/folders/1hUovlrl5aRi8b84KhHS5DOg0tzT_1JyB?usp=sharing.  You can download the checkpoint folders and put them in ./code. Then you can run
```bash
>> bash run_xxx.sh
>> bash evaluate_xxx.sh
```
to reproduce our results.
Also, for your question, I think it may be due to hardware devices or Pytorch itself.

**Comment by XXRPKU** [comment]
Thank you!
